ar-eg/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="ar">
+<head>
+  <title>ماتطلبش انك تسأل، اسأل على طول.</title>
+  <meta property="og:title" content="ماتطلبش انك تسأل، اسأل على طول." />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com/ar-eg" />
+  <meta property="og:locale" content="ar_EG" />
+  <meta property="og:image" content="https://dontasktoask.com/favicon.png"/>
+  <link rel="icon" href="/favicon.png" type="image/png" />
+  <link rel="stylesheet" type="text/css" href="../style.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <style>
+      blockquote{
+        border-right: 10px solid #CCC;
+        border-left: initial;
+      }
+      .name{
+          display: block;
+      }
+  </style>
+</head>
+<body dir="rtl">
+  <main>
+    <h1>ماتطلبش انك تسأل، اسأل على طول.</h1>
+    <p>
+        من وقت للتاني، بيجي حد ويقول:
+    </p>
+    <blockquote>
+      <span class="name">فولان:</span>
+      <p class="message">
+        حد يعرف جافا هنا؟؟
+      </p>
+    </blockquote>
+    <p>
+        الطريقة دي وحشة جدا في السؤال لاكتر من سبب. هو الشخص ده عايز يقول:
+    </p>
+    <blockquote>
+      <span class="name">فولان:</span>
+      <p class="message">
+          حد يعرف جافا هنا يساعدني في المشكلة بتاعتي اللي ممكن تطلع ملهاش علاقة بجافا اصلا او لو حد مش عارف جافا ممكن يجاوب على سؤالي؟
+      </p>
+    </blockquote>
+    <p>
+        في اسباب كتير ممكن تخلي الناس اللي عندهم خبرة مايجوبوش على سؤالك، سؤالك بيطلب اكتر ما انت فاكر.
+    </p>
+    <p>
+        انت كده بتحمل الناس مسئولية، اكنك بتختبر ثقتهم في قدراتهم.
+        
+        وكده غصب عنك انت بتخلي ناس ماتجاوبش على سؤالك، يعني انا ممكن لو فيه سؤال عن لغة برمجة انا معرفش عنها حاجة اقدر اجاوب عليه بالـ"سينس" كده.
+    </p>
+    <p>
+      بشكل تاني ممكن سؤالك يتشاف زي ما يكون:
+    </p>
+    <blockquote>
+      <span class="name">فولان:</span>
+      <p class="message">
+          انا عندي سؤال عن جافا، بس انا مكسل اكتبه بشكل واضح لحد ما الاقي حد ياخد الخطوة ويعرض انه هيساعدني.
+      </p>
+    </blockquote>
+    <p>
+        وده يعتبر كسل. لو انت مش عايز تتعب شوية عشان تكتب سؤالك بشكل كويس .. ليه احنا هنتعب ونساعدك؟
```

Comment:
Make sure that this line and the whole code is compatible with RTL (Right To Left) because the dots and the question marks might mess up the whole thing, do that by opening a live server (Hot Reload) and than checking on every line.

Also consider deleting spaces between the words and the dots after like "word ..." to "word..."

Feel free to take a look at the original Arabic code.

Reply:
Yes, I tested the RTL. check it out:
![image](https://user-images.githubusercontent.com/29176293/126045104-ed3c9b78-59ca-4c35-ad58-70d15022e3f9.png)